AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1877,7 +1892,9 @@ protected void displayCardQuestion() {
             long delay = mWaitAnswerSecond * 1000 + mUseTimerDynamicMS;
             if (delay > 0) {
                 mTimeoutHandler.removeCallbacks(mShowAnswerTask);
-                mTimeoutHandler.postDelayed(mShowAnswerTask, delay);
+                if(!mSpeakText){
```

Comment:
Same as above - whitespace changes only to match project standards

```suggestion
                if (!mSpeakText) {
```